### PR TITLE
feat(example): add ddp and crop zones metadata

### PR DESCRIPTION
**Pull request by CedrickArmel**
Add ddp and crop zones metadata.

### CODE REVIEW

**Review by Gab-pm — APPROVED** [review]

**Comment by ccamel** [comment]
@CedrickArmel The tests failed. Can you take a look and fix it? Thank you.

File: example/ddp/ddp.ttl [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@prefix core: <https://ontology.okp4.space/core/> .
+@prefix zone: <https://ontology.okp4.space/dataverse/zone/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix topic: <https://ontology.okp4.space/thesaurus/topic/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<https://ontology.okp4.space/dataverse/zone/metadata/86500dd6-080c-4565-8779-8a0aa052f166> a owl:NamedIndividual,  
+        <https://ontology.okp4.space/metadata/zone/GeneralMetadata> ;
+    core:describes zone:d74127ac-d411-4fef-9d65-73c6fc22315d ;  
+    core:hasDescription "Porté par CERFRANCE Bretagne, la data est dans le pré (ou DDP) est un data space dont l'objectif est de valoriser les données agricoles (techniques et comptables) au travers d’un mécanisme de fixation du prix juste pour les produits laitiers, pour ensuite fournir des conseils spécifiques à chaque agriculteur."@fr ;
```
Comment (ccamel) [inline example/ddp/ddp.ttl:10]: I think `data space` should be replaced by `zone`.

File: example/ddp/gouvernance.ttl [hunk 2]
```diff
@@ -0,0 +1,844 @@
...
+    metagov:hasNumber "1.2.4.1.1" ;
+    core:isPartOf govmeta:5e1d591f-c8ca-41c6-9d56-9d4098cdb1c0  .  
+
+govmeta:64462eac-3644-4306-8c01-93ba694946ba a owl:NamedIndividual,  
+        metagov:Section ; 
+    core:hasTitle "Gestion des jeux de données"@fr ;
+    core:hasTitle "Datasets Management"@en ; 
+    core:hasTitle "Verwaltung von Datensätzen"@de ;     
+    core:hasDescription "Il s'agit des règles visant à rendre les jeux de données accessibles au sein du Data Space."@fr ;
+    core:hasDescription "These are the rules for making datasets accessible within the Data Space."@en ;
```
Comment (ccamel) [inline example/ddp/gouvernance.ttl:209]: Same here. Use the term `Zone` instead.

**Review by ccamel — CHANGES_REQUESTED** [review]

**Review by Gab-pm — COMMENTED** [review]
I will check with clients if the files can be public before to approve

File: example/ddp/ddp.ttl (hunk 1 above)
Comment (CedrickArmel) [reply example/ddp/ddp.ttl:10]: @ccamel i checked the files, read the test code made several amendment commits and still don't see what still causes the test to fail.

Comment (ccamel) [reply example/ddp/ddp.ttl:10]: @CedrickArmel did you rebased against `main` after #152 merge?

**Comment by CedrickArmel** [comment]
@ccamel I have just made just made a `git rebase origin/main` but nothing changes.

**Comment by ccamel** [comment]
> @ccamel I have just made just made a `git rebase origin/main` but nothing changes.

Need to test locally to see what's going wrong, because the GitHub action logs don't provide the information (should be improved).

**Comment by ccamel** [comment]
I have identified the issue. It appears that there are 2 instances of `meta:Paragraph` with the same identifier, which results in 2 `core:hasNumber` properties with the same subject. Fortunately, this is being detected by the tests.

Fix is currently underway.

**Review by ccamel — APPROVED** [review]
LGTM!

**Comment by ccamel** [comment]
Here we are. @CedrickArmel wait for @Gab-pm decision before merging.

**Review by Gab-pm — APPROVED** [review]

**Comment by ccamel** [comment]
All right! Thanks all for your contribution.

**Pull request merged**